build.gradle

```diff
@@ -210,65 +229,86 @@ ext.libraries = [
+    }
+    compile group: 'com.sun.xml.bind', name: 'jaxb-osgi', version: '2.3.3'
+    compile group: 'io.springfox', name: 'springfox-swagger2', version: versions.springfoxSwagger
+    compile group: 'io.springfox', name: 'springfox-swagger-ui', version: versions.springfoxSwagger
 
   compile group: 'uk.gov.hmcts.reform', name: 'logging', version: versions.reformLogging
   compile group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformLogging
   compile group: 'uk.gov.hmcts.reform', name: 'properties-volume-spring-boot-starter', version: '0.1.0'
   compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: versions.reformS2sClient
 
-  compile group: 'org.flywaydb', name: 'flyway-core', version: '6.5.1'
-  compile group: 'org.postgresql', name: 'postgresql', version: '42.2.14'
-
-
-  compile group: 'com.google.guava', name: 'guava', version: '27.0-jre'
-
-  compile group: 'com.google.guava', name: 'guava', version: '28.2-jre'
-
-  testCompile group: 'io.rest-assured', name: 'rest-assured', version: '3.3.0'
+    compile group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.11.3'
+    compile group: 'io.jsonwebtoken', name: 'jjwt', version:'0.9.1'
+    compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.11.3'
+    compile group: 'javax.inject', name: 'javax.inject', version: '1'
+    compile group: 'uk.gov.hmcts.reform', name: 'idam-client', version: '1.3.1'
+    compile "org.springframework.boot:spring-boot-starter-oauth2-client"
+    compile "org.springframework.boot:spring-boot-starter-oauth2-resource-server"
+    compile group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '8.20'
+    implementation "io.github.openfeign:feign-httpclient:10.7.0"
+    implementation 'org.springframework.boot:spring-boot-starter-validation'
+
+    compile group: 'org.flywaydb', name: 'flyway-core', version: '5.2.4'
+    compile group: 'org.postgresql', name: 'postgresql', version: '42.2.14'
+
+    compile group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.13.3'
+    compile group: 'org.apache.logging.log4j', name: 'log4j-to-slf4j', version: '2.13.3'
+
+    compile group: 'com.google.guava', name: 'guava', version: '29.0-jre'
+    compile group: 'javax.el', name: 'javax.el-api', version: '3.0.0'
+
+    compile group: 'com.launchdarkly', name: 'launchdarkly-java-server-sdk', version: versions.launchDarklySdk
+    compile group:"org.yaml", name: "snakeyaml", version:"1.27"
+
+    compileOnly group: 'org.projectlombok', name: 'lombok', version: versions.lombok
+    annotationProcessor group: 'org.projectlombok', name: 'lombok', version: versions.lombok
+    testCompileOnly group: 'org.projectlombok', name: 'lombok', version: versions.lombok
+    testAnnotationProcessor group: 'org.projectlombok', name: 'lombok', version: versions.lombok
+    integrationTestCompileOnly group: 'org.projectlombok', name: 'lombok', version: versions.lombok
```

Comment:
can we keep Lombok CompileOnly hence we can get all places funcompile,intCompile and we can reduce no of lines.